<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebDialogImportFileType</class>
 <widget class="QWidget" name="WebDialogImportFileType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="subCaptionLabel">
     <property name="text">
      <string>Pick input data type to continue</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="chromatogrammsRadio">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Import chromatogramms</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="fragmentsRadio">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Import fragments</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
